### DISCUSSION TITLE
What's the difference between nav:collection:pages and collection:pages?

### DISCUSSION THREAD

**Discussion by niloc7** [statamic/cms#4766] [Q&A] [answered]
I tried using both. And both worked fine.

The docs seem to show that Nav is purely for navigation list, while Collection pulls in data from everywhere.

But technically, both can do the same thing.

So am I missing something here, like perhaps Nav is "less heavy" or something? Hmm..

**Comment by jasonvarga** [comment, marked as answer]
The `nav` tag is generally for creating a nested list - multiple levels. Or for navigations specifically. So if you're only doing a single level collection, you should stick with the `collection` tag.

**Reply by niloc7** [reply]
I see.. thanks!
